debian/ceph-mgr-dashboard.requires

```diff
@@ -0,0 +1,12 @@
+bcrypt
+CherryPy
+more-itertools
+PyJWT
+pyopenssl
+requests
+Routes
+prettytable
+pytest
+pyyaml
+natsort
+ceph-common
```

Comment:
Does this `requires.txt` format support version ranges? If yes, why not directly reading from the original `requirements.txt`? That way we would avoid duplication, which is a common source of pain in this case. If we find a way [for rpmbuild/ceph.spec to be fed](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_build_time_dependency_generator) from those, then we would improve our Python packaging for our 3 major distro families (CentOS/Fedora, Debian/Ubuntu and OpenSUSE).

Reply:
yeah, that'd be ideal. as put in https://github.com/ceph/ceph/pull/45643#discussion_r884204346 , i'd fix this piecemeal when i get a chance.